client/trino-jdbc/src/test/java/io/trino/jdbc/TestTrinoDatabaseMetaData.java

```diff
@@ -1132,8 +1132,9 @@ public void testGetTablesMetadataCalls()
                         list(COUNTING_CATALOG, "test_schema1", "test_table1", "TABLE"),
                         list(COUNTING_CATALOG, "test_schema2", "test_table1", "TABLE")),
                 new MetadataCallsCount()
-                        .withListSchemasCount(1)
-                        .withListTablesCount(2));
+                        .withListSchemasCount(3)
```

Comment:
What do we use 3 schema listing for during `databaseMetaData.getTables(COUNTING_CATALOG, null, "test_t_ble1", null)` call?